src/index.ts

```diff
@@ -0,0 +1,274 @@
+
+/*
+/ Types for request and response bodies.
+/
+/ Typings for requests are grouped into three categories, each following a common naming convention
+/ that mimics the convention used by Express.
+/   - Request Query Parameters: Typed as `xxxRequestQuery`
+/   - Request Path Parameters: Typed as `xxxRequestParams`
+/   - Request Body: Typed as `xxxRequestBody`
```

Comment:
These names are a bit implementation specific to the API and the names are not as relevant to a developer using the helper library. I think this is fine as long as we re-export these types in the helper functions library with more relevant names. Like for the `getQuote` function the types could be `GetQuoteParams` and `GetQuoteResponse` etc.

Reply:
+1. These aren't types that will necessarily be exposed to end-users (i.e., developers using the FiatConnect client-side SDK); I imagine that we'll of course want to export some of these types verbatim through the SDK, but for others, (like these ones) we can export some type alias to make it more readable/even change the shape of them if we want (though I don't know if that's actually useful).